samples/annotate/pom.xml

```diff
@@ -61,7 +61,12 @@
 		</dependency>
 		<dependency>
 			<groupId>com.sun.xml.bind</groupId>
-			<artifactId>jaxb-impl</artifactId>
+			<artifactId>jaxb-core</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
I'll take a look at this, maybe there is another fix.. I've added it, because build fails: dependency from commons-logging wasn't found